src/cloud-api-adaptor/test/e2e/assessment_runner.go

```diff
@@ -349,9 +349,9 @@ func (tc *TestCase) Run() {
 
 				if tc.podState == v1.PodRunning {
 					if len(tc.testCommands) > 0 {
-						logString, err := AssessPodTestCommands(ctx, client, tc.pod, tc.testCommands)
+						err := AssessPodTestCommands(t, ctx, client, tc.pod, tc.testCommands)
 						if err != nil {
-							t.Errorf("AssessPodTestCommands failed, with output: %s and error: %v", logString, err)
+							t.Fatal(err.Error())
```

Comment:
I don't think we want to switch to fatal here as that would be the clean-up stages are skipped